ui/components/multichain/token-list/token-list.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import classnames from 'classnames';
+import {
+  DISPLAY,
+  FLEX_DIRECTION,
+  FONT_WEIGHT,
+  JustifyContent,
+  Size,
+  TextColor,
+  TextVariant,
+} from '../../../helpers/constants/design-system';
+import {
+  AvatarNetwork,
+  AvatarToken,
+  AvatarWithBadge,
+  AVATAR_WITH_BADGE_POSTIONS,
+  Text,
+} from '../../component-library';
+import Box from '../../ui/box/box';
+import { getNativeCurrencyImage } from '../../../selectors';
+
+export const NewTokenList = ({
+  className,
+  onClick,
+  tokenSymbol,
+  tokenImage,
+  primary,
+  secondary,
+}) => {
+  const primaryTokenImage = useSelector(getNativeCurrencyImage);
+  return (
+    <Box
+      className={classnames('token-list', className)}
+      display={DISPLAY.FLEX}
+      flexDirection={FLEX_DIRECTION.COLUMN}
+      gap={4}
+      margin={4}
```

Comment:
If we do `as="a"` on each cell, they become accessible by keyboard.

Reply:
I think using `as="button"` would be better since we are adding an `onClick` prop. But I have not used this polymorphic prop here and updated it with role="button" , `onKeyPress` and `tabIndex` prop to make it more accessible. Wdyt?


https://user-images.githubusercontent.com/39872794/222803334-9286e663-9eb6-4bad-8200-3c8cdb652035.mov